src/Slider/Thumbs.tsx

```diff
@@ -0,0 +1,69 @@
+import { useRef } from "react";
+import { VisuallyHidden, mergeProps, useFocusRing, useSliderThumb } from "react-aria";
+
+const getRailWidth = (lowerInputThumbProps, higherInputThumbProps) => parseFloat(higherInputThumbProps.style?.left.toString().slice(0, -1)) - parseFloat(lowerInputThumbProps.style?.left.toString().slice(0, -1));
+
+const useThumb = (index, trackRef, name, state) => {
+  const inputRef = useRef(null);
+  const { thumbProps, inputProps } = useSliderThumb({
+    index,
+    trackRef,
+    inputRef,
+    name
+  }, state);
+  const { focusProps } = useFocusRing();
+  return {
+    inputRef,
+    thumbProps,
+    inputProps,
+    focusProps,
+  }
+}
+
+const Thumb = ({ thumbProps, inputRef, inputProps, focusProps }) => {
+  return (
+    <div
+      {...thumbProps}
+      className="thumb"
+    >
+      <VisuallyHidden>
+        <input ref={inputRef} {...mergeProps(inputProps, focusProps)} />
```

Comment:
We should aria-label this input "minimum"/"maximum" - maybe a `thumbLabel` prop that can be passed in.

I usually don't worry too much about translations in aria attributes, but if we want to enable that later, we could expose "minLabel"/"maxLabel" as props on `Slider`.

Reply:
added minimum/maximum hardcoded (for now, can maybe add a prop later)